scripts/run.js

```diff
@@ -1,22 +1,39 @@
 // Will deploy the WavePortal Contract and console the address it is deployed to
 const main = async () => {
-    const [owner, randomPerson] = await hre.ethers.getSigners();
-    const waveContractFactory = await hre.ethers.getContractFactory("WavePortal");
-    const waveContract = await waveContractFactory.deploy();
+    const {ethers} = hre;
+    const [owner, randomPerson] = await ethers.getSigners();
+    const waveContractFactory = await ethers.getContractFactory("WavePortal");
+    const waveContract = await waveContractFactory.deploy({
+        value: ethers.utils.parseEther("0.1")
+    });
     await waveContract.deployed(); // Waits for contract to be deployed
 
     console.log(`Contract deployed to ${waveContract.address}`);
     console.log(`Contract deployed by: ${owner.address}`);
 
+    // Gets contract balance
+    let contractBalance = await ethers.provider.getBalance(waveContract.address); 
+    console.log(
+        "Contract balance:",
+        ethers.utils.formatEther(contractBalance)
+    );
+
     await waveContract.getTotalWaves(); // Gets total waves and logs it to the console
 
-    const firstWaveTxn = await waveContract.wave(); // Increment totalWaves by one
+    const firstWaveTxn = await waveContract.wave('TEST'); // Increment totalWaves by one
     await firstWaveTxn.wait(1); // Wait for 1 block confirmation to make sure wave() fully ran
 
     await waveContract.getTotalWaves(); // Get total wave again and it should be one greater than before
 
+    // Tests change in balance after messages is sent
+    contractBalance = await ethers.provider.getBalance(waveContract.address); // Gets contract balance
+    console.log(
+        "Contract balance:",
+        ethers.utils.formatEther(contractBalance)
+    );
```

Comment:
You can remove the `console.log`s throughout the PR

Reply:
I have removed the console.logs throughout the contract except for those that logged errors in "try catches." Am I okay to merge this branch or do you have something else you would like me to change?